CMakeLists.txt

```diff
@@ -33,6 +33,16 @@ if (BUILD_TESTS)
     enable_testing()
 endif()
 
+add_custom_target(copy_cfg ALL
```

Comment:
did I ask about this CMakeLists.txt change? I guess it's a good idea but I don't see why this is done in this PR.

Reply:
Well i added the `copy_addons` so we can do `make cppcheck && ./bin/cppcheck --addon=...`, and I moved `copy_cfg` with it so these are defined together.